src/django/pfb_analysis/serializers.py

```diff
@@ -27,7 +27,9 @@ def get_neighborhood_label(self, obj):
 
     class Meta:
         model = AnalysisJob
-        exclude = ('created_at', 'modified_at', 'created_by', 'modified_by', 'overall_scores',)
+        exclude = ('created_at', 'modified_at', 'created_by', 'modified_by', 'overall_scores',
+                   'analysis_job_definition', 'tilemaker_job_definition',
+                   '_analysis_job_name', '_tilemaker_job_name',)
```

Comment:
Time to switch to a whitelist?  I believe it would be
```
        fields = ('uuid', 'createdAt', 'modifiedAt', 'running_time', 'start_time', 'status',
                  'neighborhood_label', 'neighborhood', 'overall_score', 'batch_job_id',
                  'osm_extract_url', 'census_block_count', 'batch')
```
Not shorter, but not longer, and whitelists are easier to read and sometimes produce fewer unwelcome surprises.